src/pages/registration/files_and_license_tab/FilesTableRow.tsx

```diff
@@ -275,17 +275,20 @@ export const FilesTableRow = ({
                           row
                           sx={{ flexWrap: 'nowrap' }}
                           onChange={(event) => {
-                            const fileVersion = event.target.value as FileVersion;
-                            setFieldValue(field.name, fileVersion);
+                            const newFileVersion = event.target.value as FileVersion;
+                            const previousFileVersion = field.value;
+                            setFieldValue(field.name, newFileVersion);
 
                             if (isRrsApplicableCategory) {
-                              if (fileVersion === FileVersion.Published) {
+                              if (newFileVersion === FileVersion.Published) {
```

Comment:
`previousFileVersion ` vil nå alltid lages som en const, men vil bare brukes om man kommer inn i denne if-en. Så jeg ville vurdert å enten flytte consten inn dit hvor den trengs for å unngå å lage variabler som ikke vil brukes, eller bare bruke verdien direkte

```suggestion
                              if (field.value === FileVersion.Published) {
```